config/development.json

```diff
@@ -93,8 +93,9 @@
 		"upgrades/domain-management/transfer": true,
 		"upgrades/domain-search": true,
 		"upgrades/free-trials": true,
+		"upgrades/cart/multi-year-domain": true,
 
-		"manage/customize": true,
+	  "manage/customize": true,
```

Comment:
You've used spaces here - it will look weird for anyone that has configured `tab` to be expanded to something else than 2 spaces :)